galsim/phase_psf.py

```diff
@@ -523,39 +533,37 @@ def illuminated(self):
         """
         return self._illuminated
 
-    @property
+    @lazy_property
     def rho(self):
         """ Unit-disk normalized pupil plane coordinate as a complex number:
         (x, y) => x + 1j * y.
         """
-        if not hasattr(self, '_rho') or self._rho is None:
-            u = np.fft.fftshift(np.fft.fftfreq(self.npix, self.diam/self.pupil_plane_size/2.0))
-            u, v = np.meshgrid(u, u)
-            self._rho = u + 1j * v
-        return self._rho
+        f1 = np.fft.fftfreq(self.npix, 1./self.pupil_plane_size)
```

Comment:
That's up to you I guess.  There are a few functions (these plus some others) that aren't used anywhere, but I added simple unit tests for them, since I thought maybe they were still useful for debugging purposes.  But if you think you won't need them anymore, feel free to zot them.  (I removed the unused f1, f2 calculations at least.)